### PR TITLE
(WIP) JSC use case

### PR DESCRIPTION
**Pull request by damianam**
There you go. Still have to check some things to get correct info in some points (the internet connection on the train is not too reliable for ssh'ing and checking), but most of what you need is here already.

### CODE REVIEW

File: docs/users/hpc-ugent.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ HPC-UGent (Belgium)
 
 :website: http://www.ugent.be/hpc
 
-:institute: Ghent University - http://www.ugent.be
+:institution: Ghent University - http://www.ugent.be
```
Comment (boegel) [inline docs/users/hpc-ugent.rst]: please also change this in `cscs.rst` to keep things consistent?

File: docs/users/jsc.rst [hunk 2]
```diff
@@ -3,64 +3,128 @@
...
 ============================================
 
-:website: http://www.fz-juelich.de/ias/jsc
+:website: http://www.fz-juelich.de/jsc
 
-:institute: Forschungszentrum Jülich - http://www.fz-juelich.de/portal/EN
+:institution: Forschungszentrum Jülich - http://www.fz-juelich.de/portal/EN
 
-:contact: *(to be completed)*
+:contact: Damian Alvarez (d.alvarez@fz-juelich.de)
```
Comment (boegel) [inline docs/users/jsc.rst]: maybe having multiple contacts here makes sense in your case? include Alan/Markus?

File: docs/users/jsc.rst [hunk 3]
```diff
@@ -3,64 +3,128 @@
...
-*(last update: Oct 23rd 2016)*
+*(last update: Feb 24th 2017)*
 
 .. _users_use_cases_jsc_history:
 
 History
 -------
 
-*(to be completed)*
+The first contact between JSC and EasyBuild was through the LinkSCEEM (TODO: add url) project in 2013. During that period the
```
Comment (boegel) [inline docs/users/jsc.rst]: http://www.eumedgrid.eu/index.php/linksceem

File: docs/users/jsc.rst [hunk 4]
```diff
@@ -3,64 +3,128 @@
...
+* Operating system: CentOS Linux 7
+* Processor architecture: Intel Haswell
+* Number of nodes: 1872
+* GPUs: NVIDIA K80 and K20x
+* Interconnects: InfiniBand
+
+JUROPA3
+~~~~~~
+* Operating system: Scientific Linux 7 (TODO: verify)
+* Processor architecture: Intel Sandy Bridge, Intel KNL, Intel KNC
```
Comment (boegel) [inline docs/users/jsc.rst]: use `Intel Xeon Phi Knights Landing`, `Intel Xeon Phi Knights Corner`?

File: docs/users/jsc.rst [hunk 5]
```diff
@@ -3,64 +3,128 @@
...
-*(to be completed)*
+At the moment, EasyBuild is used in JSC'S main production system and a 2 internal systems
+
+JURECA
+~~~~~~
+* Operating system: CentOS Linux 7
+* Processor architecture: Intel Haswell
+* Number of nodes: 1872
+* GPUs: NVIDIA K80 and K20x
+* Interconnects: InfiniBand
```
Comment (boegel) [inline docs/users/jsc.rst]: include link to http://www.fz-juelich.de/ias/jsc/EN/Expertise/Supercomputers/JURECA/JURECA_node.html?

Comment (boegel) [inline docs/users/jsc.rst]: also, which IB generation?

File: docs/users/jsc.rst [hunk 6]
```diff
@@ -3,64 +3,128 @@
...
+| gmvolfc        | 2016b-GDR                 | 5.4.0          | GCC 5.4.0        | MVAPICH2 2.2-GDR       | 8.0.44 | OLF 2016b      | GCCcore, GCC, gmvapich2c      | Compiler+MPI+CUDA+Math toolchain                         |
+| imvmklc        | 2016b-GDR                 | 5.4.0          | Intel 2016.4.258 | MVAPICH2 2.2-GDR       | 8.0.44 | MKL 2017.0.098 | GCCcore, iccifort, imvapich2c | Compiler+MPI+CUDA+Math toolchain                         |
+| pmvmklc        | 2016b-GDR                 | 5.4.0          | PGI 16.9         | MVAPICH2 2.2-GDR       | 8.0.44 | MKL 2017.0.098 | GCCcore, PGI, pmvapich2c      | Compiler+MPI+CUDA+Math toolchain                         |
++----------------+---------------------------+----------------+------------------+------------------------+--------+----------------+-------------------------------+----------------------------------------------------------+
+
+*OLF 2016b: OpenBLAS 0.2.19, LAPACK 3.6.1, ScaLAPACK 2.0.2, FFTW 3.3.5
```
Comment (boegel) [inline docs/users/jsc.rst]: what's this bit?

File: docs/users/jsc.rst [hunk 7]
```diff
@@ -3,64 +3,128 @@
...
 
 Configuration
 ~~~~~~~~~~~~~
 
-*(to be completed)*
+EasyBuild is configured using a ``Developers`` module that sets up correctly all the EasyBuild related environment
+variables
+
+TODO: put the output of the command
```
Comment (boegel) [inline docs/users/jsc.rst]: ideally, provide output of `eb --show-config`?

File: docs/users/jsc.rst [hunk 8]
```diff
@@ -3,64 +3,128 @@
...
 Notes
 ~~~~~
 
-*(none)*
+* JSC deploys software in "Stages". A stage is snapshot of the latest versions of all the available software at a given
+  time. The stages change every 6 months, with the arrival of new projects.
+* There is a development stage for testing software before moving it to production
+* The software is deployed in a shared GPFS filesystem
+* Software is deployed in production just by the infrastructure manager
+* The module cache is updated by cron job every 30 minutes
```
Comment (boegel) [inline docs/users/jsc.rst]: missing `.`s ?

also, include link to http://hpcugent.github.io/easybuild/files/eb-jsc-hust16.pdf?

File: docs/users/jsc.rst (hunk 6 above)
Comment (damianam) [inline docs/users/jsc.rst]: What do you mean?

Comment (boegel) [inline docs/users/jsc.rst]: Ah, after checking it again I noticed that this `OLF` line is a footnote, I missed that initially.

**Pull request closed without merging**